app/src/main/java/nu/yona/app/ui/settings/SettingsFragment.java

```diff
@@ -222,7 +219,7 @@ public void onError(Object errorMessage) {
                 }
             });
         } catch (Exception e) {
-            AppUtils.throwException(SettingsFragment.class.getSimpleName(), e, Thread.currentThread(), null);
+            AppUtils.reportException(SettingsFragment.class.getSimpleName(), e, Thread.currentThread(), null);
             showAlert(e.toString(), false);
```

Comment:
Remove the ``showAlert`` here. Deletion cannot fail because of a user error, so don't report it tot the user that way.